omero/developers/Matlab.txt

```diff
@@ -100,6 +99,27 @@ but it is not configurable from within MATLAB::
     c.enableKeepAlive(60); % Call session.keepAlive() every 60 seconds
     c.closeSession();      % Close session to end the keep-alive
 
+Working in a different group
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Each session is created within a given context defining not only the session
```

Comment:
I think that would be wise.


On 23 Sep 2015, at 11:36, Sébastien Besson <notifications@github.com<mailto:notifications@github.com>> wrote:


In omero/developers/Matlab.txt<https://github.com/openmicroscopy/ome-documentation/pull/1282#discussion_r40189551>:

> @@ -100,6 +99,27 @@ but it is not configurable from within MATLAB::
>      c.enableKeepAlive(60); % Call session.keepAlive() every 60 seconds
>      c.closeSession();      % Close session to end the keep-alive
>
> +Working in a different group
> +^^^^^^^^^^^^^^^^^^^^^^^^^^^^
> +
> +Each session is created within a given context defining not only the session


Just pushed. Should we invoke @hflynn<https://github.com/hflynn>'s call on the punctuation question?

—
Reply to this email directly or view it on GitHub<https://github.com/openmicroscopy/ome-documentation/pull/1282/files#r40189551>.